modules/special-rules/pages/vortex-missile.adoc

```diff
@@ -9,11 +9,12 @@ The vortex consumes anything it contacts in the brief moments before it collapse
 
 ---
 
-When you shoot with a {vortex-missile}:
+When you shoot a {vortex-missile}:
```

Comment:
It might help to know that I chose 'with' for a reason: It is good practice to avoid possible verb confusion. "Shoot a..." can mean either "Shoot with a..." or "Shoot at a..." (possibly that you miss) or "Shoot dead a..." (you hit).

English readers will mostly get the intended meaning of your change just fine, but even then it can cause the mind to do a double-take and stumble on it. For a non-native English reader it can actually be confusing or at least make it harder to parse.

That said, I'm thinking that I might change it to "When you make a {vortex-missile} shot:" -- which would mirror the construction we discussed for `death-ray.adoc` and which I've started to implement in some of the other weapons.

Reply:
Aye.